data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/file/FileWatcher.java

```diff
@@ -38,19 +38,22 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * This class is responsible for watching a given file and reports update.
+ * This class is responsible for watching a given file and reports update or execu
```

Comment:
` or execu`?

Reply:
emm weired. Seems like spotless cuts it.